esiclient/tests/functional/README.md

```diff
@@ -5,7 +5,7 @@ These tests are designed to ensure Ironic and Neutron behave as expected when co
 ### Prerequisites
 
 These tests are intended to be ran against a functioning OpenStack cloud with Ironic and Neutron services enabled and running. It assumes that the ironic.conf file
-is located at `/etc/ironic/ironic.conf`. These tests assume that the `fake-hardware` hardware type is enabled under the `enabled_hardware_types` option under `[DEFAULT]` heading.
+is located at `/etc/ironic/ironic.conf`. These tests assume that the `fake-hardware` hardware type is enabled under the `enabled_hardware_types` option under `[DEFAULT]` heading. Some of these tests require an Ironic node configured with ipmi credentials in order to test the use of the ipmitool-socat serial console.
```

Comment:
May as well link to the Ironic documentation page describing what's needed.

Reply:
added